framework/scripts/agent_upgrade.py

```diff
@@ -225,4 +226,4 @@ def main():
     except Exception as e:
         print(f"Internal error: {str(e)}")
         if args.debug:
-            raise
+            raise
```

Comment:
Add a blank line at the end
```suggestion
            raise

```